packages/backend/src/services/SavedSemanticViewerChartService/SavedSemanticViewerChartService.ts

```diff
@@ -275,36 +321,12 @@ export class SavedSemanticViewerChartService extends BaseService {
             savedSemanticViewerChartUuid,
         );
 
-        const { hasAccess: hasUpdateAccess } = await this.hasSavedChartAccess(
-            user,
+        await this.hasAccess(
             'update',
-            savedChart,
+            { user, projectUuid },
+            { savedSemanticViewerChartUuid },
         );
 
-        if (!hasUpdateAccess) {
-            throw new ForbiddenError(
-                "You don't have permission to update this chart",
-            );
-        }
-
-        // check permission if the chart is being moved to a different space
-        if (
-            update.unversionedData &&
-            savedChart.space.uuid !== update.unversionedData.spaceUuid
```

Comment:
was this check redundant? I'm not sure exactly what it does but cannot see the same check in the `checkAccess`

Reply:
yes! that was checking an access to a new space (target space) that is being done in `hasAccess` now.